code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -360,14 +360,12 @@
 
 /datum/greyscale_config/labcoat
 	name = "Labcoat"
-	icon_file = 'icons/mob/clothing/suits/labcoat.dmi'
-	json_config = 'code/datums/greyscale/json_configs/labcoat_worn.json'
+	icon_file = 'icons/obj/clothing/suits/labcoat.dmi'
+	json_config = 'code/datums/greyscale/json_configs/labcoat.json'
 
 /datum/greyscale_config/labcoat/worn
 	name = "Labcoat (Worn)"
 	icon_file = 'icons/mob/clothing/suits/labcoat.dmi'
```

Comment:
```suggestion
	icon_file = 'icons/mob/clothing/suits/labcoat.dmi'
	
```
everything checks out, just uh-
need to keep this whitespace between the config and the comment below (Can't post a Suggestion on deleted lines so I'm doing it here)